### ISSUE TITLE
I broke it

### ISSUE THREAD

**Issue by Decilitre** [tensorflow/playground#208]
![image](https://github.com/user-attachments/assets/eaa4490e-feb2-4406-980b-d3b0893df130)

**Comment by MimoHasPurpose** [comment]
how did u break it? @CoderUnkn0wn

**Comment by Decilitre** [comment]
You can see in the image all of the settings I used. Then I just let it run and suddenly everything was NaN.

**Comment by MimoHasPurpose** [comment]
coooll

**Comment by TuchoTurco** [comment]
this is not an issue, it is normal behavior.

**Comment by frozenfrank** [comment]
## Second Witness
"Breaking" the model by causing the weights to blow up to infinity isn't difficult to do. Setting the learning rate high at the start of a complex model causes the weights to explode to high values, evaluate to Infinity, and then the next epoch reports them all as "NaN." 

## Discussion
I think it is normal behavior— it's just the way the math works out. Setting the learning rate so high is bound to give inaccurate results.

This kind of playground is the perfect place to be able to play with settings like that and experience how the models behave when given diverse kinds of inputs. The researcher student will quickly learn to be more intentional about the use of a high learning rate!

## Minimal Reproduction
I was able to reproduce the behavior previously reported by doing the following:
1. Set the learning rate to the maximum
2. Max out the number of hidden layers and hidden nodes
3. Set the problem type to "regression"
4. Step through the training one epoch at a time
5. Discover the values overflowing within **14 training epochs**

### Walkthrough

#### Epoch 1: Loss stats already reporting a value with 46 zeros behind it
<img width="1329" alt="Screenshot 2024-09-30 at 10 56 52 PM" src="https://github.com/user-attachments/assets/46c6093e-a232-4d9e-b558-e2c0515e0ffb">

#### Epoch 7: Loss stats overflowing and reporting "Infinity"
<img width="1287" alt="Screenshot 2024-09-30 at 10 57 36 PM" src="https://github.com/user-attachments/assets/9c29c753-615c-4eac-9a88-ae7bc25655f0">

#### Epoch 14: Loss stats reporting `NaN`
<img width="1284" alt="Screenshot 2024-09-30 at 10 58 01 PM" src="https://github.com/user-attachments/assets/00a3cb6c-f52e-4580-a48b-60f0873d9710">
